Homework_4/seminar_task_7.py

```diff
@@ -0,0 +1,17 @@
+"""✔ Функция получает на вход словарь с названием компании в качестве ключа
+и списком с доходами и расходами (3-10 чисел) в качестве значения.
+✔ Вычислите итоговую прибыль или убыток каждой компании. Если все компании
+прибыльные, верните истину, а если хотя бы одна убыточная — ложь.
+"""
+
+
+def are_profitable(companies: dict[str:list[int]]):
+    return all(0 if i < 0 else i for i in map(sum, companies.values()))
```

Comment:
Решение верное, но можно чуть проще.
```suggestion
    return all(map(lambda x: sum(x) > 0, companies.values()))
```